tests/test_create.py

```diff
@@ -293,3 +294,47 @@ def test_create_untyped_dict() -> None:
 
     cfg = DictConfig(ref_type=Dict, content={})
     assert get_ref_type(cfg) == Optional[Dict]
+
+
+def test_yaml_duplicate_keys() -> None:
```

Comment:
1. do not merge the tests. two different behaviors gets two tests. (one for duplicate keys, one for merge support).
2. looks like you have two tests for duplicate keys. Can you parameterize with pytest.mark.parametereize?